### ISSUE TITLE
Invalid paths in lmms shared object files

### ISSUE THREAD

**Issue by karimuscormaven** [LMMS/lmms#3749]
```
[lmms]# uname -a
Linux 4.11.12-200.fc25.i686+PAE #1 SMP Fri Jul 21 17:00:30 UTC 2017 i686 i686 i386 GNU/Linux

[lmms]# 

Name        : lmms-vst
Source RPM  : lmms-1.1.3-3.fc24.src.rpm
URL         : https://lmms.io/
Summary     : VST hosting plugin for lmms

Name        : lmms
Source RPM  : lmms-1.1.3-3.fc24.src.rpm
URL         : https://lmms.io/


The following entries details the error in LMMS to find
its EFFECT libraries.

[lmms]# pwd
/usr/lib/lmms
[lmms]# 

[lmms]# ls
ladspa \                              libkicker.so                libpapu.so                          libvestige.so
libamplifier.so                     libladspabrowser.so  libpatman.so                      libvibedstrings.so
libaudiofileprocessor.so     libladspaeffect.so      libpeakcontrollereffect.so   libvstbase.so
libbassbooster.so              liblb302.so                 libsf2player.so                    libvsteffect.so
libbitinvader.so                  libmalletsstk.so          libsfxr.so                             libwatsyn.so
libdelay.so                         libmidiimport.so         libsid.so                              libwaveshaper.so
libdualfilter.so                    libmonstro.so            libspectrumanalyzer.so       libZynAddSubFxCore.so
libdynamicsprocessor.so   libnes.so                   libstereoenhancer.so          libzynaddsubfx.so
libflpimport.so                   libOPL2.so                libstereomatrix.so
libhydrogenimport.so        liborganic.so             libtripleoscillator.so
[lmms]# 

## The missing (.so) IS located in the /usr/lib/lmms dir, but the (.so) has the wrong path !
## The path /usr/lib/vst_base DOES NOT EXIST ! As shown above the (.so) files are in the
## path /usr/lib/lmms  !!!
## The package was installed via (dnf) automatically -- this is not a manual install !

[lmms]# ldd -v libvsteffect.so 
	linux-gate.so.1 (0xb7721000)
	../vst_base/libvstbase.so => not found               <------ERROR
	libstdc++.so.6 => /lib/libstdc++.so.6 (0xb754d000)
	libm.so.6 => /lib/libm.so.6 (0xb74f6000)
	libgcc_s.so.1 => /lib/libgcc_s.so.1 (0xb74d8000)
	libc.so.6 => /lib/libc.so.6 (0xb72fe000)
	/lib/ld-linux.so.2 (0xb7723000)

[root@tc-prime3 lmms]# ldd -v libvestige.so 
	linux-gate.so.1 (0xb7793000)
	../vst_base/libvstbase.so => not found               <-----ERROR
	libstdc++.so.6 => /lib/libstdc++.so.6 (0xb75b4000)
	libm.so.6 => /lib/libm.so.6 (0xb755d000)
	libgcc_s.so.1 => /lib/libgcc_s.so.1 (0xb753f000)
	libc.so.6 => /lib/libc.so.6 (0xb7365000)
	/lib/ld-linux.so.2 (0xb7795000)

[lmms]# 
```

**Comment by tresf** [comment]
We don't package`rpm` files.  This is a Fedora/dnf problem.  In the meantime... Try this instead... 


https://github.com/LMMS/lmms/pull/3688

**Comment by karimuscormaven** [comment]
Thanks TresI will repost at Fedora Project.I appreciate the timely response !
 "When change is bearing down on you - "Jump to the next track !" 

    On Thursday, August 3, 2017 8:48 PM, Tres Finocchiaro <notifications@github.com> wrote:
 

 Closed #3749.—
